functions/src/modules/pm/categories/prizes/updatePrize.ts

```diff
@@ -1,5 +1,58 @@
-import { ExpressFunction } from "../../../../@types";
 
-const replaceMe: ExpressFunction = (req, res, next) => {};
 
-export default replaceMe;
+import type { ExpressFunction } from "../../../../@types";
+import { firebaseApp } from "../../../../config/firebaseConfig";
+import {
+  expressErrorHandlerFactory,
+  requestBodyTypeValidator,
+} from "../../../../helpers";
+import { uasPermissionSwitch } from "../../../../systems/uas";
+
+
+const updatePrize: ExpressFunction = (req, res, next) => {
+  uasPermissionSwitch({
+    organizer: { accepted: ValidateOrganizer },
+  })(req, res, next);
+};
+
+
+const ValidateOrganizer: ExpressFunction = (req, res, next) => {
+  const validationRules: ValidatorObjectRules = {
+    type: "object",
+    rules: {
+      prizeDisplayName: { rules: ["string"],  },
+      prizePrice: { rules: ["string"],  },
+      prizeUrl: { rules: ["string"]},
+      prizeasin: { rules: ["string"]},
+      prizeListingName: { rules: ["string"], },
+      prizeurl: { rules: ["string"], },
```

Comment:
You've got URL twice...
```suggestion
     },
```